packages/wonder-blocks-form/src/components/text-field.md

```diff
@@ -1,15 +1,91 @@
-A demonstration of a default TextField.
+Text
 
 ```js
 import {TextField} from "@khanacademy/wonder-blocks-form";
 
-<TextField />
+class TextFieldExample extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: "",
+        };
+    }
+
+    render() {
+        return (
+            <TextField
+                id="tf-1"
+                type="text"
+                value={this.state.value}
+                onChange={(newValue) => this.setState({value: newValue})}
+            />
+        );
+    }
+}
+
+<TextFieldExample />
+```
+
+Number
+
+```js
+import {TextField} from "@khanacademy/wonder-blocks-form";
+
+class TextFieldExample extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: "",
+        };
+    }
+
+    render() {
+        return (
+            <TextField
+                id="tf-1"
+                type="number"
+                value={this.state.value}
+                onChange={(newValue) => this.setState({value: newValue})}
+            />
+        );
+    }
+}
+
+<TextFieldExample />
+```
+
+Password
+
+```js
+import {TextField} from "@khanacademy/wonder-blocks-form";
+
+class TextFieldExample extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: "",
```

Comment:
Same with this story.

https://www.chromatic.com/pullrequest?activeElementId=comparison-test-key-60ae86565fbe3da84802e7cd-1200&appId=5e1bf4b385e3fb0020b7073c&number=1126&view=changes